src/cmd-buildupload

```diff
@@ -71,27 +70,27 @@ def s3_upload_build(args):
                 args.enable_gz_peel):
             nogz = bn[:-3]
             img['path'] = nogz
-            s3_cp(args, path, f'{buildid}/{nogz}',
+            s3_cp(args, path, f'{dest}/{nogz}',
```

Comment:
I'm confused by the split between `builddir` and `dest` here.  It looks like `dest` won't have the architecture?  Or I guess that changes later?

Reply:
>  Or I guess that changes later?

Yup, exactly!

#580 has:


```python
if builds.is_legacy():
    s3_upload_build(args, builds.get_build_dir(args.build), args.build)
else:
    for arch in builds.get_build_archs(args.build):
        s3_upload_build(args, builds.get_build_dir(args.build, arch),
                        f'{args.build}/{arch}')
```

so we'll indeed be uploading with the same layout in the bucket.